rotkehlchen/chain/evm/decoding/ens/decoder.py

```diff
@@ -0,0 +1,321 @@
+import logging
+from abc import abstractmethod
+from typing import TYPE_CHECKING
+
+import content_hash
+
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.assets.utils import TokenEncounterInfo, get_or_create_evm_token
+from rotkehlchen.chain.ethereum.abi import decode_event_data_abi_str
+from rotkehlchen.chain.ethereum.modules.ens.constants import (
+    ADDR_CHANGED,
+    CONTENT_HASH_CHANGED,
+    NEW_OWNER,
+    NEW_RESOLVER,
+    TEXT_CHANGED_KEY_AND_VALUE,
+    TEXT_CHANGED_KEY_AND_VALUE_ABI,
+    TEXT_CHANGED_KEY_ONLY,
+    TEXT_CHANGED_KEY_ONLY_ABI,
+)
```

Comment:
these constants should be at `evm/decoding/ens/constants.py` or what do you think?